full/src/test/java/apoc/mongodb/MongoDBTest.java

```diff
@@ -68,6 +68,10 @@
 
     private static final long NUM_OF_RECORDS = 10_000L;
 
+    private static List<ObjectId> refs;
+    private static ObjectId nameAsObjectId = new ObjectId("507f191e810c19729de860ea");
+    private static ObjectId idAsObjectId = new ObjectId();
+
     @BeforeClass
     public static void setUp() throws Exception {
         assumeFalse(isTravis());
```

Comment:
is there a reason for this restriction?

Reply:
@fbiville I'm answering on this, because I did it, we don't want to execute container tests on Travis because they took to much time with the current Travis resources